test/fixtures/mockUpdate.js

```diff
@@ -0,0 +1,21 @@
+const update0 = require('./g-train-0');
+const update1 = require('./g-train-1');
+const stopInterval = 10
+
+module.exports = function mockUpdate(callCount = 0) {
```

Comment:
Maybe I'm missing something but it seems like this call count approach will only work properly for one requestor. For better predicability and the ability to have multiple clients connect to a single test server, what about requesting a specific update file using a query parameter (using an enum like "?version=DEFAULT" / "?version=FIRST_UPDATE") instead?

I don't really care what the actual enums are, but I think it would be great to use more descriptive names for the files than 0 and 1 and to generally match those file names to the enums we use.  We can discuss those names if you want but only if you agree to the approach first :)